### PR TITLE
Update documentation for bind9 to be consistent with wheel installation

### PR DESCRIPTION
**Pull request by kozzie**
### What does this PR do?
Update documentation for bind9 to be consistent with wheel installation

### Motivation
What inspired you to submit this pull request?
Noticed that the documentation was incorrect while helping a customer with the setup.
Zendesk: #221506

### CODE REVIEW

File: bind9/README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ The Bind9 check is **NOT** included in the [Datadog Agent](https://app.datadoghq
 To install the Bind9 check on your host:
 
 1. Install the [developer toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit) on any machine.
-2. Run `ddev release build neo4j` to build the package.
+2. Run `ddev release build bind9` to build the package.
 3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
-4. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w <BIND9_DIST_PATH>/<ARTIFACT_NAME>.whl`.
+4. Upload the build artifact to any host with an Agent and run `sudo -u dd-agent -- datadog-agent integration install -w path/to/bind9/dist/<ARTIFACT_NAME>.whl`.
```
Comment (l0k0ms) [inline bind9/README.md]:
```suggestion
4. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w <BIND9_DIST_PATH>/<ARTIFACT_NAME>.whl`.
```

**Comment by l0k0ms** [comment]
Hello, many thanks for you contribution and for spotting those glitches. 🙇 Just added a small nit to follow the placeholder logic.

**Comment by kozzie** [comment]
@l0k0ms thanks for checking over this for me!

`datadog-agent integration install -w <BIND9_DIST_PATH>/<ARTIFACT_NAME>.whl` doesn't work for me. Are you sure this is correct?

**Comment by l0k0ms** [comment]
@kozzie what's the issue popping out? A sudo permission error?

**Pull request merged**